### DISCUSSION TITLE
How to make world origin to be bound to map's corner rather then playerstart?

### DISCUSSION THREAD

**Discussion by Maxilos** [microsoft/AirSim#3773] [Support Q&A] [answered]
After I updated airsim recently, I found out that the world origin is now being set to the playerstart position, which is not very helpful in my case. What changes do I need to make to revert  world origin to being on the map's corner?

**Comment by jonyMarino** [comment, marked as answer]
Hi and welcome @Maxilos! You can comment this line:
`this->GetWorld()->SetNewWorldOrigin(FIntVector(player_loc) + this->GetWorld()->OriginLocation);`
in https://github.com/ahmed-elsaharti/AirSim/blob/master/Unreal/Plugins/AirSim/Source/SimMode/SimModeBase.cpp

**Reply by Maxilos** [reply]
Thanks a lot, it worked! Just to be sure, it shouldn't create any other problems, right?

**Reply by jonyMarino** [reply]
Now the semantics of the APIs are going to be different for you. Because if you pass a position to a method(like in moveToPosition() API), it will be a position relative to the world center and not to where the pawn has started. That is the only (respectable) thing.
